devices/milight.coffee

```diff
@@ -50,6 +50,8 @@ module.exports = (env) ->
       r = Number("0x#{newColor[1..2]}")
       g = Number("0x#{newColor[3..4]}")
       b = Number("0x#{newColor[5..6]}")
+      if r == 255 && g == 255 && b == 255
```

Comment:
Why you like set it to withe mode in this case? In my opinion white in color mode and white mode are two different things.